spec/controllers/api/v1/rents_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+describe Api::V1::RentsController do
+  include_context 'Authenticated User'
+
+  describe 'GET #index' do
+    let!(:rents) { create_list(:rent, 3, user: user) }
+
+    before { get :index, params: { user_id: user.id } }
+
+    it { expect(response).to have_http_status(:ok) }
+
+    it 'responds with the user\'s rents in json format' do
+      expected = ActiveModel::Serializer::CollectionSerializer.new(
+        rents, each_serializer: RentSerializer, root: false
+      ).to_json
+
+      expect(response_body.to_json) =~ JSON.parse(expected)
```

Comment:
Doesn't it work with `.to eq`?

Reply:
Try using `expect(response_body).to have_been_serialized_with(RentSerializer)`

Reply:
@lmasello @facucompe I couldn't make it work with any of the two approaches.  It could be that I'm missing some extra code so that `have_been_serialized_with` works (a gem or a helper maybe?).

The closest thing I found was this https://robots.thoughtbot.com/validating-json-schemas-with-an-rspec-matcher. Should I do it that way?

Reply:
It's also a nice approach. However, instead of adding another gem let's take a look at the contents of `response_body`. 

If we insert a debugger (e.g., `byebug`) before this expect, we see that `response_body` contains the expected result but as a value of the key `page` (we may guess that this is because of the pagination, right?).

So, something like: 
`expect(response_body['page'].to_json).to eq expected` should work as well as facu's suggested approach. You can use whichever you like :)

Reply:
Thanks! My bad. I didn't check the response properly. I will use the `.to  eq` approach. I did like the `have_been_serialized_with` approach, but I asked around and turns out it's a Wolox custom matcher.